### PR TITLE
Update Windows pre-reqs article for 3.0 release

### PR DESCRIPTION
**Pull request by mairaw**
Contributes to #14217 

I still need to add the screenshots for VS 2019 but you can review the text already

### CODE REVIEW

File: docs/core/windows-prerequisites.md [hunk 1]
```diff
@@ -46,18 +46,51 @@ For download links and more information, see [.NET downloads](https://dotnet.mic
 >
 > In addition to KB2999226, make sure you also have [KB2533623](https://support.microsoft.com/help/2533623/microsoft-security-advisory-insecure-library-loading-could-allow-remot) installed. If you don't have this update installed, you'll see an error similar to the following when you launch a .NET Core application: `The library hostfxr.dll was found, but loading it from C:\<path_to_app>\hostfxr.dll failed`.
 
-## Prerequisites for .NET Core 3.0 Preview 3
+## Prerequisites with Visual Studio
```
Comment (jamshedd) [inline docs/core/windows-prerequisites.md]: Is "with" the right word here?

File: docs/core/windows-prerequisites.md [hunk 2]
```diff
@@ -46,18 +46,51 @@ For download links and more information, see [.NET downloads](https://dotnet.mic
...
+    
+You can use any editor to develop .NET Core applications using the .NET Core SDK. Visual Studio provides an integrated development environment for .NET Core apps on Windows.
```
Comment (jamshedd) [inline docs/core/windows-prerequisites.md]: Should we qualify Visual Studio as "Visual Studio 2017 or later". That's protection against someone assuming an older version like VS 2015 could be used for Core development.

**Review by jamshedd — CHANGES_REQUESTED** [review]

**Comment by mairaw** [comment]
Thanks @jamshedd. I've addressed your feedback.

**Comment by mairaw** [comment]
I have a related question. If you try to target .NET Core 3.0 in a VS version that isn't supported, you get a build error with the error code NETSDK1045. 

That error message doesn't have any topic associated with a doc right now:
https://docs.microsoft.com/en-us/visualstudio/index?f1url=https%3A%2F%2Fmsdn.microsoft.com%2Fquery%2Fdev16.query%3FappId%3DDev16IDEF1%26l%3DEN-US%26k%3Dk(NETSDK1045)%26rd%3Dtrue&view=vs-2019

Should I add that keyword to this topic?

**Comment by jamshedd** [comment]
> I have a related question. If you try to target .NET Core 3.0 in a VS version that isn't supported, you get a build error with the error code NETSDK1045.
> 
> That error message doesn't have any topic associated with a doc right now:
> https://docs.microsoft.com/en-us/visualstudio/index?f1url=https%3A%2F%2Fmsdn.microsoft.com%2Fquery%2Fdev16.query%3FappId%3DDev16IDEF1%26l%3DEN-US%26k%3Dk(NETSDK1045)%26rd%3Dtrue&view=vs-2019
> 
> Should I add that keyword to this topic?

Yes you can, additionally I would consider pointing a user that gets the error to the SDK version selection article: https://docs.microsoft.com/en-us/dotnet/core/versions/selection

File: docs/core/windows-prerequisites.md [hunk 3]
```diff
@@ -46,18 +46,51 @@ For download links and more information, see [.NET downloads](https://dotnet.mic
...
+
+To develop .NET Core apps in Visual Studio 2019 using the .NET Core 3.0 SDK:
+
+ 1. [Download and install Visual Studio 2019 version 16.3.0 or higher](/visualstudio/install/install-visual-studio) with the **.NET Core cross-platform development** workload (in the **Other Toolsets** section) selected.
+
+<!-- ![Screenshot of Visual Studio 2019 installation with the ".NET Core cross-platform development" workload selected](./media/windows-prerequisites/vs-2017-workloads.jpg) -->
+
+Visual Studio 2019 16.3 uses .NET Core 3.0 SDK by default after the workload is installed.
+
+If you want your existing projects to use the latest .NET Core runtime, retarget existing .NET Core projects to .NET Core 3.0 using the following instructions:
```
Comment (rpetrusha) [inline docs/core/windows-prerequisites.md]: Perhaps make it clearer that this is a per-project setting (e.g., retarget each existing .NET Core project...)

**Review by rpetrusha — APPROVED** [review]
I've left a couple of comments for you to consider, @mairaw.

**Comment by Thraka** [comment]
@mairaw Did you get a screenshot? I was doing this same update but I can revert mine since yours seems more detailed. 

Screenshot I was going to use: https://github.com/dotnet/docs/blob/2bb2107901735265b8c0958bbfbd9d4dd5de56c1/docs/core/media/windows-prerequisites/vs-2019-workloads.png

**Comment by mairaw** [comment]
Yes, I did add a screenshot. Similar to the one we had before:
https://github.com/dotnet/docs/blob/625d95a99165c12edb5088ac2be944c667a967d8/docs/core/media/windows-prerequisites/vs-2019-workloads.jpg

**Comment by mairaw** [comment]
I was ready to merge this. If you want, you can help with the macOS version. Let me know.

**Comment by Thraka** [comment]
My PR did some stuff to the macOS article. I'll remove the window article from my PR

**Pull request merged**